src/main/kotlin/no/risc/risc/RiScService.kt

```diff
@@ -145,8 +145,39 @@ class RiScService(
                             RiScStatus.SentForApproval, RiScStatus.Draft -> githubConnector::fetchDraftedRiScContent
                         }
                         fetchRisc(owner, repository, id.id, accessTokens.githubAccessToken.value)
+                            .let {
+                                when (id.status) {
+                                    RiScStatus.Draft -> {
+                                        /*
+                                        * Because of our unusual datastorage, this is an extra state check for Drafts.
+                                        *
+                                        * In case a repository does not delete branches after merging pull requests,
+                                        * our state flowchart would go from SentForApproval (pull request) to
+                                        * Draft (branch exists). Therefore we check if the content in Draft is equal to
+                                        * the content on the default branch (often main). If they are equal then we
+                                        * know we have a ros branch without changes, and therefore it should be in a
+                                        * Published state.
+                                        *  */
+
+                                        // Get ros content on default branch.
+                                        val published = githubConnector.fetchPublishedRiSc(owner, repository, id.id, accessTokens.githubAccessToken.value)
+
+                                        val fileFound = published.status === GithubStatus.Success
```

Comment:
Hva representerer Github-statusene? Hvordan kallet mot Github gikk for denne riscen?

Reply:
Riktig 😊  Det er statusen på requesten til github 👍